### PR TITLE
Compatible with pyinstaller --windowed mode

### LINKED ISSUE #60: Cant use this if build app use pyinstaller --windowed mode
Hi, thanks for this project.

I found that I can't use this module(version 3.1.0) if my app is builded by pyinstaller --windowed. I can run my app directly with command "python app.py".

I catch the exception:

File "ftpsync\util.py", line 104, in <module>
OSError: [WinError 6] The handle is invalid

That line 104 in ftpsync\util.py is "IS_REDIRECTED = os.fstat(0) != os.fstat(1)", I change that to "IS_REDIRECTED = False" for temporary workaround.

### PR DESCRIPTION
**Pull request by AnyOfYou**
Fix #60

### CODE REVIEW

File: ftpsync/util.py [hunk 1]
```diff
@@ -157,7 +157,10 @@ def check_cli_verbose(default=3):
 
 DEFAULT_CREDENTIAL_STORE = "pyftpsync.pw"
 DRY_RUN_PREFIX = "(DRY-RUN) "
-IS_REDIRECTED = os.fstat(0) != os.fstat(1)
+try:
+    IS_REDIRECTED = os.fstat(0) != os.fstat(1)
+except OSError:
+    IS_REDIRECTED = False
```
Comment (mar10) [inline ftpsync/util.py:163]: Could you add a short comment here with a reference to the issue?

**Review by mar10 — CHANGES_REQUESTED** [review]
Thanks for contributing. Please also add an entry to the changeling

File: ftpsync/util.py (hunk 1 above)
Comment (AnyOfYou) [reply ftpsync/util.py:163]: Sure thing

**Review by mar10 — APPROVED** [review]
thanks

**Pull request merged**